CSS/BoardView-Body.css

```diff
@@ -17,6 +17,10 @@
     max-width: 500px;
 }
 
+.card:hover {
+    cursor: pointer;
+}
+
```

Comment:
This does not affect the title but the whole list.

Reply:
If you want to make a pointer cursor on the title of a list do as shown below: 
![image](https://user-images.githubusercontent.com/52054023/99919615-84fcdc00-2cec-11eb-9c04-1424f9fdbf3c.png)

Reply:
Good catch - however, moving it within `card-Title` only changes the mouse icon when we hover over the line of text - I've now moved it to `card-header`, which covers the entire card.

Reply:
The modal only appears when you click on the card-title.

Reply:
Oh, just noticed this - will make the change now 👍